src/main/java/com/axibase/tsd/client/SelfSignedTrustManager.java

```diff
@@ -1,18 +1,21 @@
 package com.axibase.tsd.client;
 
+import lombok.extern.slf4j.Slf4j;
+
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.TrustManagerFactory;
 import javax.net.ssl.X509TrustManager;
+import java.io.Serializable;
 import java.security.KeyStore;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.security.KeyStore'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AUselessImportCheck)